scrapy/mail.py

```diff
@@ -4,17 +4,13 @@
 See documentation in docs/topics/email.rst
 """
 import logging
-
-try:
-    from cStringIO import StringIO as BytesIO
-except ImportError:
-    from io import BytesIO
 import six
 
 from email.utils import COMMASPACE, formatdate
 from six.moves.email_mime_multipart import MIMEMultipart
 from six.moves.email_mime_text import MIMEText
 from six.moves.email_mime_base import MIMEBase
+from six import BytesIO
```

Comment:
If I'm not missing something, we need to preserve cStringIO here as well

Reply:
Sorry, missed that, thanks for correcting me.